### PR TITLE
Save Databricks init scripts in the workspace [databricks]

### PR DESCRIPTION
**Pull request by NvTimLiu**
To close: https://github.com/NVIDIA/spark-rapids/issues/8916

Init scripts in DBFS will be deprecated sept 1 2023.

Move Databricks init scripts in the workspace

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
build

File: jenkins/databricks/clusterutils.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def generate_create_templ(sshKey, cluster_name, runtime, idle_timeout,
             for path in path_list:
                 templ['init_scripts'].append(
                     {
-                        'dbfs' : {
+                        'workspace' : {
```
Comment (tgravescs) [inline jenkins/databricks/clusterutils.py:55]: needs copyright update to file

File: jenkins/databricks/init_cuda11_runtime.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #
 
 # The init script to install cuda11.0 toolkit
-# Will be automatically pushed into the dbfs:/databricks/init_scripts once it is updated.
+# Will be automatically pushed into the Databricks workspace: /databricks/init_scripts once it is updated.
```
Comment (tgravescs) [inline jenkins/databricks/init_cuda11_runtime.sh:19]: needs copyright update

**Comment by NvTimLiu** [comment]
Oh, I'd like to move DB workspace create and clean up into the common functions, let me try it.

File: jenkins/Jenkinsfile-blossom.premerge-databricks [hunk 3]
```diff
@@ -129,7 +129,7 @@ String getDbType() {
 void databricksBuild() {
     def CLUSTER_ID = ''
     def SPARK_MAJOR = BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS.replace('.', '')
-    def dbfs_path = "$INIT_SCRIPTS_DIR-$DB_TYPE"
+    def ws_path = "$INIT_SCRIPTS_DIR-$DB_TYPE"
```
Comment (gerashegalov) [inline jenkins/Jenkinsfile-blossom.premerge-databricks:132]: another way of doing this is to add spark-rapids github repo under /Repos. Then we can reference an initscript path /Repos/user@email/spark-rapids/...

Comment (NvTimLiu) [reply jenkins/Jenkinsfile-blossom.premerge-databricks:132]: If we create a mirror spark-rapids GitHub repo on Databricks /Repos/user@email/spark-rapids dir, it is inflexible to point the init_script files at the PR for the pre-merge pipeline, or pointing to someone's private GitHub repo for testing pipelines?

**Review by pxLi — APPROVED** [review]
LGTM to unblock usage after Sep. 1

If any follow-ups, lets handle in new PRs, thanks~

**Review by jlowe — APPROVED** [review]

**Pull request merged**